core/src/main/java/com/linecorp/armeria/internal/common/metric/RequestMetricSupport.java

```diff
@@ -343,6 +353,22 @@ public Counter writeTimeouts() {
         public Counter responseTimeouts() {
             return responseTimeouts;
         }
+
+        @Override
+        public DistributionSummary retries() {
```

Comment:
Based on @trustin 's comments, I guess we need two variants of retries: `successRetries()`, `failureRetries` each with a tag `result: success|failure`. We may determine whether a retry has been successful via `successFunction`.

ref: https://github.com/line/armeria/pull/4707#discussion_r1119776879

Reply:
Thanks a lot for your suggestions,
So I think we need to use `Counter` instead of `DistributionSummary` here as we need to check whether a retry has been successful and do increment by 1.

Reply:
We can continue using `DistributionSummary`. You can just create two `DistributionSummary`s, one for successful ones and the other for failed ones. They will just have two different meter IDs whose names are same and tags are different.